.travis.yml

```diff
@@ -6,6 +6,7 @@ node_js:
   - 0.10
   - 0.12
   - 4.0
+  - 6.0
```

Comment:
Please revert this change as it's not obvious why this would change with a version bump of a dependency -  feel free to open up a new PR in order to get tests for node.js 6.x :+1: